flimfitgen.py

```diff
@@ -34,8 +35,8 @@ def usage():
     "/MCR_R2015b_maci64_installer.zip"
 
 for x, y in (
-        ("FLIMFIT_51_WIN", "artifacts/FLIMfit_@VERSION@_x64.zip"),
-        ("FLIMFIT_51_MAC", "artifacts/FLIMfit_@VERSION@_MACI64.zip"),
+        ("FLIMFIT_52_WIN", "artifacts/FLIMfit_@VERSION@_x64.zip"),
+        ("FLIMFIT_52_MAC", "artifacts/FLIMfit_@VERSION@_MACI64.zip"),
```

Comment:
I changed the filename to match these instead when I was trying to test this, but I can certainly amend them if you want to give me a new filename.